### PR TITLE
Tests/packaging: Remove CentOS/RHEL 7 specific content

### PR DESCRIPTION
**Pull request by tyll**
Signed-off-by: Till Maas <opensource@till.name>

Remove all CentOS7/RHEL specific content (most importantly the CI), so we can focus on fixing bugs only on the supported platforms.

---
This change is [Reviewable](https://reviewable.io/reviews/nmstate/nmstate/574)

### CODE REVIEW

File: automation/README.md [hunk 1]
```diff
@@ -3,17 +3,9 @@ The automation env is serving the tests of Nmstate.
 It may be used both locally and through CI.
 
 ## Components
-- Dockerfile: Defines a Centos 7 based container image which includes systemd,
-  NetworkManager and other basic build tools (e.g. tox).
-
-  The image can be found at:
-  https://hub.docker.com/r/nmstate/centos7-nmstate-dev/
-
-- Dockerfile.fedora: Defines a Fedora based container image for Nmstate test
```
Comment (EdDev) [inline automation/README.md]: Why do you remove this?

File: automation/README.md [hunk 2]
```diff
@@ -55,8 +47,8 @@ You may change the test type by using:
  * `./automation/run-tests.sh --test-type unit_py37`:
    Unit tests in Python 3.7 using 'nmstate/fedora-nmstate-dev' docker image.
 
- * `./automation/run-tests.sh --test-type integ --el7`:
-   Integration tests using 'nmstate/centos7-nmstate-dev' docker image.
+ * `./automation/run-tests.sh --test-type integ --el`:
```
Comment (EdDev) [inline automation/README.md]: What happened to the `8`?

File: automation/run-tests.sh [hunk 3]
```diff
@@ -237,7 +237,7 @@ while true; do
         shift
         TEST_TYPE="$1"
         ;;
-    --el7)
+    --el)
```
Comment (EdDev) [inline automation/run-tests.sh]: I think that the `8` emphasizes what it is.

File: packaging/README-dockerhub.md [hunk 4]
```diff
@@ -1,16 +1,13 @@
 ## Docker Hub
 
-The images are automatically rebuilt on new GIT tags or pushes to the master branch:
+The images are automatically rebuilt on new GIT tags or pushes to the master branch:^
```
Comment (EdDev) [inline packaging/README-dockerhub.md]: What is the `^` doing there?

File: automation/README.md (hunk 1 above)
Comment (tyll) [reply automation/README.md]: The container specifications are in https://github.com/nmstate/nmstate/tree/master/packaging and the file is called differently there.

File: automation/README.md (hunk 2 above)
Comment (tyll) [reply automation/README.md]: @cathay4t changed it like this in the other PR, as long as we are not supporting multiple enterprise Linuxes, the version does not make sense IMHO.

File: automation/run-tests.sh (hunk 3 above)
Comment (tyll) [reply automation/run-tests.sh]: Changed it to `--el8`

File: automation/README.md (hunk 2 above)
Comment (tyll) [reply automation/README.md]: changed it to `--el8`.

**Pull request merged**